dmaws/commands/provision.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+import subprocess
+
+from ..cli import cli_command
+from ..utils import run_cmd
+
+
+def run_playbook(playbook, hosts, ctx, basedir='playbooks/'):
+    args = [
+        'ansible-playbook',
```

Comment:
Yeah. And we're gathering the variables in python and sending the full dictionary to ansible, so it's only responsible for actually running the provisioning tasks